app/views/physical_ticket/show.pdf.prawn

```diff
@@ -0,0 +1,64 @@
+prawn_document(filename: @file_name, page_layout: :portrait, :page_size =>'A4' ) do |pdf|
```

Comment:
you have the `pdf` view, what about the normal show? should look similar to the pdf! Could be in aother PR, but has to be done!